pkg/apply/event/event.go

```diff
@@ -72,6 +72,7 @@ const (
 	PruneAction
 	DeleteAction
 	WaitAction
+	InventoryAction
```

Comment:
Does any of the cli code (in particular the printing code) make assumptions about which type of events they should expect? We probably should avoid them failing on unknown event types, but I'm not sure how it is done today.

Reply:
Currently, it just does nothing because the switch statement for inventory event types do not have any code:

https://github.com/kubernetes-sigs/cli-utils/blob/master/pkg/print/list/base.go#L129

We can add extra verbosity inventory printing, but I don't think we need them to start.